src/go/rpk/pkg/cli/cmd/container/start.go

```diff
@@ -38,6 +42,22 @@ type node struct {
 	addr string
 }
 
+type clusterPorts struct {
+	KafkaPorts     []uint
```

Comment:
(still seeing public in envVarTemplateData)

Reply:
Right, I remember it now. So envVarTemplateData cannot be private because `text/template` requires the struct fields to be exported in order for it to read it.